spec/intl.html

```diff
@@ -1692,8 +1692,8 @@ <h1>Temporal.Calendar.prototype.monthDayFromFields ( _fields_ [ , _options_ ] )<
             1. If _calendar_.[[Identifier]] is *"iso8601"*, then
               1. Let _result_ be ? ISOMonthDayFromFields(_fields_, _options_).
             1. Else,
-              1. Set _fields_ to ? PrepareTemporalFields(_fields_, « *"day"*, *"era"*, *"eraYear"*, *"month"*, *"monthCode"*, *"year"* », « *"day"* »).
               1. Let _overflow_ be ? ToTemporalOverflow(_options_).
+              1. Set _fields_ to ? PrepareTemporalFields(_fields_, « *"day"*, *"era"*, *"eraYear"*, *"month"*, *"monthCode"*, *"year"* », « *"day"* »).
```

Comment:
Fair enough, I buy that arguments should be processed in order. (though I'm not sure we did that consistently in other methods)

The current change didn't require a polyfill update because the polyfill already (mistakenly) worked like that, but I'll update the polyfill when I reverse the order in all six places.

Reply:
If not, that's something that ideally would have been caught in stage 3 review - arguments should always be processed (modulo legacy methods) in order.

Reply:
I never saw that documented anywhere, but it seems like a good thing to document for spec writers, as it removes a potential avenue for observable discrepancies between spec text sections that are authored by different people. If it's documented somewhere, can you point me to it? I'll open an issue to audit all the existing methods for this.

Reply:
I don't think we've noted it down anywhere, but it's repeatedly come up in plenary (eg, around `Object.hasOwn` argument order being swapped from `Object.prototype.hasOwnProperty`) and we've consistently preferred argument order.

Reply:
Do we have an existing document in how-we-work that lists these kinds of things? I couldn't see one at a quick glance. Maybe a new doc called `editorial-conventions.md` would be a good place for this?

Reply:
https://github.com/tc39/ecma262/issues/2236 tracks those, but this is a normative convention, so it's a bit different - there's not really a place for it yet.

Reply:
Ideally it should be written down somewhere ASAP — it potentially would've saved me a lot of work if I'd known about it two years ago, or if stage 3 reviewers had known to watch out for it. I'll see if I can take a stab at a document this week.

Reply:
→ https://github.com/tc39/how-we-work/pull/119